x-pack/plugin/downsample/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/downsample/10_basic.yml

```diff
@@ -445,7 +445,7 @@ setup:
           index.blocks.write: true
 
   - do:
-      catch: /index_not_found_exception/
+      catch: /index_not_found_exception|illegal_argument_exception/
```

Comment:
Maybe add a comment on top of this api call to explain why we either expect of the two exceptions?

Reply:
Done.